### DISCUSSION TITLE
How to get a Profile Pair Extraordinaire?

### DISCUSSION THREAD

**Discussion by solidm568** [community/community#56569] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How can I get a Profile Pair Extraordinaire? Please help me :)

**Comment by SolidMaster568** [comment, marked as answer]
This is one way to do it

create a branch and commit changes
get a collaborator to commit changes on the same branch
create a pull request
merge to default branch using squash and merge

**Reply by solidm568** [reply]
Thanks, you save my time.
It worked

**Comment by tuves** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answering and locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
